csfieldguide/templates/interactives/matrix-simplifier.html

```diff
@@ -0,0 +1,173 @@
+{% extends interactive_mode_template %}
```

Comment:
Now rapidly removing matrices results in a bunch of stuff being displayed for a couple frames:
![out](https://user-images.githubusercontent.com/25916475/64213305-8e8bff80-cf00-11e9-8dd2-64176606aaab.gif)


Still:
![image](https://user-images.githubusercontent.com/25916475/64212954-3d2f4080-ceff-11e9-8f5d-af9f3b34db31.png)

It isn't a big deal so not worth putting much into, but perhaps a 'remove all' might be useful?